### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_Keyboard : The dropdown icon is not visible for keyboard users in the left navigation section

### ISSUE THREAD

**Issue by jadhavprathmesh01** [microsoft/vscode-docs#5120]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Keyboard;#WCAG2.1.1; #FTP; #ChromiumEdge;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Launch the application Visual Studio Code Services
2. Tab navigate to "Docs" link.
3. Left and right navigation will appear.
4. Tab navigate to the Left navigation links and observe that the dropdown icon is visible or no using keyboard.

**Actual:**
There are dropdowns in the "Left navigation" section and the dropdown icon is not visible when the user navigates to the links using keyboard, when we hover using mouse

**Expected Result:**
There should be dropdown icon visible when the keyboard user navigates to the links.

**User Impact:**
The keyboard only users will face difficulty if the dropdown icon is not visible.

**Attachment:**
![5120_A11y_Visual Studio Code Services_Docs_Left Right Navigation_Keyboard](https://user-images.githubusercontent.com/87858881/151154454-f7775e70-1c39-4741-819c-4f13827d5459.gif)

**Comment by gregvanl** [comment]
Duplicate of https://github.com/microsoft/vscode-docs/issues/5101

**Comment by sspatil123** [comment]
![image](https://user-images.githubusercontent.com/85863999/217260020-c40e9c83-a198-470a-aaec-f44295726728.png)
GithubTags:#Closed;Version 1.75
